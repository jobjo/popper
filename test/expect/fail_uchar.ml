open Popper
open Sample.Syntax

let () =
  Clean.run
  @@ test (fun () ->
       let* _ =
         Sample.(with_log "uchar" (Comparator.pp Comparator.uchar) uchar)
       in
       fail "Failed")
