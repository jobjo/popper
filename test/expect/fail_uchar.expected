
0/1 tests passed and 1 failed in XXs:

  ✖  Anonymous  Failed after 1 sample

┌────────────────────────────────────────────────────────────────────────────────────────┐
│ Failed Anonymous after 1 sample and 3 shrinks.                                         │
└────────────────────────────────────────────────────────────────────────────────────────┘

  Reason:

    Failed

  Log:

    uchar = \u{0}

Test Failed: 0/1 tests passed and 1 failed in XXs.

